### ISSUE TITLE
New schedule and frequency of community meetings

### ISSUE THREAD

**Issue by martinfleis** [geopandas/community#12]
Hi @geopandas/collaborators and friends, 

we have decided to change the frequency and time of community meetings. They will now happen every last Thursday of each month, instead of every other month.

We will also alter between 17:00 UTC and 20:00 UTC in an attempt to find a bit more friendly time slot for people in Australia.

The upcoming meetings (updated August 2024):

- August 29, 20:00 UTC
- September 26, 17:00 UTC
- October 31, 20:00 UTC
- November 28, 17:00 UTC

You can subscribe to the calendar to link all events to your calendar: https://calendar.google.com/calendar/u/0?cid=MjhhbTQzczBjdnRldm9sdjg4YnJybTE2Z29AZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ 

Also note that the bi-weekly meetings focusing on sub-projects as dask-geopandas are now cancelled and will be re-introduced for GSoC.

**Comment by martinfleis** [comment]
@geopandas/collaborators Due to ongoing FOSS4G, the next community meeting was moved from Thursday 25/8 to Thursday 1/9. The time 20:00 UTC remains unchanged.

**Comment by jorisvandenbossche** [comment]
And the upcoming meeting this week September 29 is moved one hour later, to 18:00 UTC

**Comment by martinfleis** [comment]
The meeting that was supposed to happen last week is happening tomorrow, Thursday 1, at 1800UTC.

**Comment by nicholas-ys-tan** [comment]
@martinfleis 

I noticed the community meetings are still listed in calendar for 2024, can I confirm if they are still ongoing? Couldn't find them referenced elsewhere.

**Comment by martinfleis** [comment]
@nicholas-ys-tan Yes, they are still ongoing as per the calendar linked in the first post. I should probably update it above...

We'd be happy to see you there, though I understand that it is not the most friendly time of a day for someone in Australia. We are switching between 1700 UTC and 2000 UTC, where the latter is somewhat doable for @m-richards and everyone else. But given the geographical distribution of maintainers, it is super tricky to find a time that suits everyone.

**Comment by m-richards** [comment]
@nicholas-ys-tan and anyone else who may be interested this months meeting will move forward to 20/6 at 20:00 UTC (same time a week earlier) due to maintainer availability, sorry for the shorter notice on the changed timeslot.

**Comment by jorisvandenbossche** [comment]
Small note that the meeting of this monthly, according the schedule for next week Thursday, has been moved to this week Thursday (October 24, 20:00 UTC). The google calendar has been updated.

**Comment by martinfleis** [comment]
The December meeting will not happen due to the holiday season. The next one is scheduled for 30/1. Calendar has been updated accordingly.
